avaframe/in3Utils/generateTopo.py

```diff
@@ -104,11 +104,11 @@ def inclinedplane(cfg):
         # make half sphere shaped channel with radius given by channel horizontal extent
         mask = np.zeros(np.shape(yv))
         mask[np.where(abs(yv) < c_extent)] = 1
-        if cfg['TOPO'].getboolean('topoconst'):
-            zv = zv - c_extent*c_0*np.sqrt(np.abs(1. - (np.square(yv) / (c_extent**2))))*mask
-        else:
+        if cfg['TOPO'].getboolean('topoadd'):
```

Comment:
Please stick to our naming convention, i.e.
```suggestion
        if cfg['TOPO'].getboolean('topoAdd'):
```

Maybe this is an opportunity to finally change all the ones that are in the Ini file, i.e c_extent to cExtent and so on...